mail-client/alpine/alpine-2.22.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="An easy to use text-based based mail and news client"
+HOMEPAGE="http://alpine.x10host.com/alpine/ https://repo.or.cz/alpine.git/"
+SRC_URI="http://alpine.x10host.com/alpine/release/src/${P}.tar.xz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine pam passfile smime spell ssl threads"
+
+DEPEND=">=sys-libs/ncurses-5.1:0=
+	kerberos? ( app-crypt/mit-krb5 )
+	ldap? ( net-nds/openldap )
+	pam? ( sys-libs/pam )
+	spell? ( app-text/aspell )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+"
+
+HTML_DOCS=("doc/tech-notes/")
```

Comment:
This gets defined, but `einstalldocs` isn't called ever in this ebuild. Do these files get installed? Perhaps you want to tie installing them to `doc` USE flag?

Reply:
This is still undone. You can just do: 
```
src_install() {
    use doc && local HTML_DOCS=( "doc/tech-notes/" )
    einstalldocs
...
```

and it should install those files and work like it's supposed to. If you do it this way, remember to delete the `HTML_DOCS` here.

Reply:
Why did you nuke the extra docs now? What's wrong with the solution given above?

Reply:
They're still installed, see line 79. I thought my solution was shorter than your proposal.